src/allotropy/parsers/utils/pandas.py

```diff
@@ -185,6 +185,8 @@ def get(
                     if raw_value is None
                     else ("true" if str_to_bool(str(raw_value)) else "")
                 )
+            if type_ is float and isinstance(raw_value, str) and "%" in raw_value:
```

Comment:
Special handling for percent values that have "%" at the end, saw one case of this